### ISSUE TITLE
rename image topics

### ISSUE THREAD

**Issue by floweisshardt** [ipa320/softkinetic#4]
please publish:
- a colored camera image (sensor_msgs/Image) under 'rgb/image_raw'
- camera infos for colored camera image (sensor_msgs/CameraInfo) under 'rgb/camera_info'
- a gray-scale depth camera image (sensor_msgs/Image) under 'depth/image_raw'
- camera infos for gray-scale depth camera image (sensor_msgs/CameraInfo) under 'depth/camera_info'

see kinect topics

**Comment by corot** [comment]
Done on indigo;  we have now:
- rgb/image_color
- rgb/image_mono
- rgb/camera_info
- depth/image_raw
- depth/camera_info
- depth/points

**Comment by gavanderhoorn** [comment]
@corot: I think I asked this before somewhere, but does that naming take existing conventions into account? @ipa-fmw suggests a `rgb/image_raw` fi, which I don't see in your list.

**Comment by corot** [comment]
Well... not fully sure if I followed the conventions, but I used the openni_launch topics as reference, just replicating the ones that hold for the softkinetic.

I assumed that, given we have now two topics on rgb (color and mono), but no rectified images, it makes more sense to have image_color and image_mono rather than image_raw. In fact, for adding an image_raw topic I would just duplicate image_mono.

**Comment by gavanderhoorn** [comment]
The thing is: _you_ know we have no rectified images, the average user doesn't (immediately). Using an established topic name (such as `image_raw`) would convey that information, without having to spend time investigating documentation and / or code.

That is why conventions / standards exist: if something claims it adheres to a standard or follows some convention, it increases ease of use (experience with other implementors of the standard can be transferred to this particular instance, etc).

The fact that it is the same as `image_mono` does not matter, imo.

**Comment by corot** [comment]
You are right, let's duplicate image_mono or image_color. I suppose image_color makes more sense, as is the direct output from the camera.
